datadog_lambda/tracing.py

```diff
@@ -67,6 +66,37 @@
 LOWER_64_BITS = "LOWER_64_BITS"
 
 
+def _extract_context(context_json, event_type, arn):
```

Comment:
I'm thinking we should update the name of this method to be a bit more specific. Now that I see it implemented, let's find a better name. I'm thinking something like `_extract_context_with_datastreams` or `_extract_context_and_dsm`.